checkov/terraform/checks/resource/aws/GlueSecurityConfigurationEnabled.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any
+
+from checkov.common.models.consts import ANY_VALUE
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+
+
+class GlueSecurityConfigurationEnabled(BaseResourceValueCheck):
+    def __init__(self) -> None:
+        name = "Ensure Glue component has a security configuration associated"
+        id = "CKV_AWS_195"
+        supported_resources = ("aws_glue_crawler", "aws_glue_dev_endpoint", "aws_glue_job")
+        categories = (CheckCategories.ENCRYPTION,)
```

Comment:
```suggestion
        categories = (CheckCategories.ENCRYPTION)
```

Reply:
if you don't add the `,` at the end, Python doesn't know that it should be a tuple with one item.

Reply:
got ya